ortools/linear_solver/xpress_interface.cc

```diff
@@ -2031,8 +2037,13 @@ void XPRS_CC optimizermsg(XPRSprob prob, void* data, const char* sMsg, int nLen,
       /* Ignore other messages */
       case 2: /* dialogue */
       case 1: /* information */
-        printf("%*s\n", nLen, sMsg);
-        break;
+      {
+        if (auto log_handler = xprs->solver_log_handler(); log_handler) {
+          log_handler->message(sMsg);
+        }
+      }
+      // printf("%*s\n", nLen, sMsg);
```

Comment:
Use this as the default behavior ? Allow the user to print logs in `std::cout` easily
```cpp
if (/* ... */; log_handler) {
 // ...
} else {
    printf(/* ... */);
}
```